first_problem_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test/unit'
+require_relative'./first_problem'
+
+class FirstProblemTest < Test::Unit::TestCase
+
+  def test_compute_empty_array_0
+    array = Array.new
```

Comment:
Usually unit tests have 3 regions: `arrange`, `execute` and `assert`, is usually a good idea to have those show in the test.

(nothing to do for now just sharing insights)